src/aioca/__main__.py

```diff
@@ -0,0 +1,16 @@
+from argparse import ArgumentParser
+
+from . import __version__
+
+__all__ = ["main"]
+
+
+def main(args=None):
+    parser = ArgumentParser()
+    parser.add_argument("-v", "--version", action="version", version=__version__)
+    args = parser.parse_args(args)
```

Comment:
If you keep the `test_cli.py` that copier made then this will be covered

Reply:
Added in https://github.com/DiamondLightSource/aioca/pull/52/commits/afd88d5602d6d944cafef0450d1577bbd6e335fd